gateway/1.4/modules/ROOT/pages/flex-gateway-k8-overview.adoc

```diff
@@ -24,8 +24,10 @@ include::partial$task-reg-run-flex-gateway.adoc[tag=k8s-connected-intro]
 == Local Mode with Kubernetes
 include::partial$task-reg-run-flex-gateway.adoc[tag=k8s-local-intro]
 
-== See Also
+The following diagram demonstrates multiple Flex Replicas deployed as ingress controllers. The Flex Replicas sits behind a load balancer and manage all incoming traffic. 
```

Comment:
```suggestion
The following diagram demonstrates multiple Flex Replicas deployed as ingress controllers. The Flex Replicas sit behind a load balancer and manage all incoming traffic. 
```

Reply:
Should "Replica" be lowercase? Or is it a proper noun?

Reply:
The capitalization is correct. It's a proper noun, or at least we refer to it as one.